src/CoreLocation/CLLocationManager.cs

```diff
@@ -47,5 +47,24 @@ public static bool IsMonitoringAvailable (Type t)
 			return false;
 		}
 #endif
+
+		public static CLAuthorizationStatus Status {
+			get {
+				// because an instance property in iOS 14, tvOS 14, Watch 7 and macOS 10,16
```

Comment:
there might be a reason why it was changed to an instance ?
if so then only exposing the static one might be problematic
if not then it might be an apple bug and should be reported

Reply:
Looking at the documentation, it makes sense to be using the instance one because the status now depends on the user interaction from the delegate and its method:
```objc
- (void)locationManagerDidChangeAuthorization:(CLLocationManager *)manager;
```
The issue here is that we need to keep the static one or we will have a breaking API change.

Reply:
We'll have to use a different name for the instance version if we want both.

Reply:
Exactly, we could do the following:

```csharp
		[Deprecated (PlatformName.iOS, 14, 0, message: "Use the instance property 'AuthorizationStatus' instead.")]
		[Deprecated (PlatformName.MacOSX, 10, 16, message: "Use the instance 'AuthorizationStatus' property instead.")]
		[Deprecated (PlatformName.TvOS, 14, 0, message: "Use the instance property AuthorizationStatus' instead.")]
		[Deprecated (PlatformName.WatchOS, 7, 0, message: "Use instance property 'AuthorizationStatus' instead.")]
		[Static]
		[Export ("authorizationStatus")]
		CLAuthorizationStatus Status { get; }

		[Watch (7, 0), TV (14, 0), Mac (10, 16), iOS (14, 0)]
		[Export ("authorizationStatus")]
		CLAuthorizationStatus AuthorizationStatus { get; }
```

That would fix the API breaking change,  I don't think we would want to add a #if XAM_CORE since we will have to support the old versions too. With this we remove the manual code, what do you think?